### DISCUSSION TITLE
文章內容層級的安排是否只限定於「中深度」？

### DISCUSSION THREAD

**Discussion by moojing** [lun-dao/LunDAO#23] [General]
今天第一次看到這個專案，覺得蠻有理想的，長期來看可以促成台灣這邊生態社群的發展，幫助大家更快可以了解區塊鏈的技術面。但這邊也想要提出一點跟文章深度有關的話題來跟大家做討論，

我在 Readme 裡面看到這個專案的原由：

> ...
> 在加密貨幣已經逐漸讓更多人熟悉的今日，網路上也有更多的新聞與文章來帶領完全不了解的群眾進入加密貨幣的世界，這些內容滿足了初階的入門者踏入這個領域的需求。但往稍微深入一些的方向前進，中文文章的資源就會逐漸減少...

我可以理解中深度的知識需要有人去做研究，理解後轉化成文章，但是我也認為既然我們要發展成 DAO ，而不是由少數幾人做為寫手的組織，就必須要考慮到社群的多元性。

如果要求所有人都只能撰寫中深度文章，對寫手的招募是不利的，因為這個領域有能力去閱讀中深度知識的人，我猜想還不多，可能只有固定的某一群，更不用說英文閱讀能力和寫作能力了，這樣長期下來極有可能社群內只會留下特定某些人。

我目前初步的建議是**在組織目標上還是可以強調以促成更多中深度文章出現為主**，但是不要把那些初級寫手拒於門外：

- 建立文章分級制度：可能一開始可以分為初、中級，但**獎勵的比例以中級為多**，吸引初級寫手藉由留在社群內藉由跟比較進階的寫手/開發者交流，慢慢自己也有能力越寫越進階。
- 必須要討論出一個文章深度的分界，例如：何謂「中深度」？界線盡可能明確，比較不會有爭議

希望現在提出這個問題不會太早，或是太過 detail ，如果會太早討論這部分的話，也許也可以留著後面給 Token Holder 們討論。

**Comment by yurenju** [comment]
主要的原因還是中深度的文章實在太少了。如果現在看網路上的文章時，最多的文章通常是把新手帶入區塊鏈的系列文章，如如何使用 MetaMask、如何使用 Compound, Uniswap 等。再往深一些是 Compound 的運作原理、如何處理治理就比較少人寫了，再更深的是 Compound 有什麼缺點，哪些專案提出了新的想法、他們的原理是什麼這些就是非常稀少。

所以這是當初寫 README 的時候考量的問題，就是越深的文章越少人寫，因為這些文章要撰寫需要很多時間消化吸收，事情又很難敘述的清楚，整體的成本就上升很多。但是這樣的文章對於 Ethereum 社群是很有價值的，這樣才可以讓大家的知識往前開拓。

但我也覺得你講得有道理是我們應該要也可以接受比較初階的文章，但用獎勵的方式調整不同深度的文章所收到的獎勵。覺得這也會有助於社群的成長，因為有了初階的文章寫作者才可以慢慢的往更深入的文章發展。

至於文章分級問題，我目前的想像是請文章作者提供一些指標，比如說
- 這篇的目標讀者是誰
- 這篇文章的主題領域是什麼
- 其他指標

用這些指標來分配這篇文章應該要給怎樣的獎勵。我知道 CryptoCow 有類似的機制，會再請教他們是怎麼做的。

**Reply by ChihChengLiang** [reply]
可見獎勵標準那段 https://medium.com/cryptocow/cryptocow-project-release-cddbce9c1ad6

**Reply by moojing** [reply]
好像可以參考一下：

> 形式的部分則會考量耗費的心力，以及對讀者的幫助做分級。第一級是針對特定主題的系統性整理，並以教學或詳細介紹/討論的形式呈現。第二級是對事件的整理，譬如爭議 EIP 的各方論述，或者 conference 的重點整理。第三級則是其餘一般的貢獻形式，包括文章翻譯、簡介、摘要、影片字幕等。

另外我覺得指標的方式不錯，也許也可以請撰文者列出閱讀文章所需要的前置知識，來作為指標之一。

**Reply by quesalltime** [reply]
有了guideline指標作為文章的分級參考之後，
下一步要考慮的應該就是由誰來審這件事。

「由誰審」這件事，可能有幾種做法

第一種：
由專案的初始參與人或是社群整體成員，選出大家認為qualified的審核委員，
再由審核委員對文章進行分級的認定與最終決斷。
至於審核委員的選出及任期方式，可再另外設計。

第二種：
由社群整體成員對文章進行分級的認定與最終決斷。

前者的好處是，文章分級的決定過程會相對有效率，
後者的好處是，讓社群整體成員對於LunDAO專案的核心目標，有更直接的參與過程，
兩者各有優劣。

**Reply by yurenju** [reply]
還有另外一個要考量的是「作者的人緣」，但這問題我也不知道是大是小，比如說有一個作者他人緣非常好，他的朋友如果持有代幣就都幫他投了，這樣也是讓整個社群投的問題之一。

對了由誰審我想要到 #35 討論耶（或是另外開一個 discussion），因為就主題是否限於中深度文章我覺得我們已經有一個共識了，想說不同主題可以另開。

**Reply by quesalltime** [reply]
了解，這件事跟發布流程息息相關，我把這個討論內容移到那邊去。

**Reply by yurenju** [reply]
這邊也不用刪，等你那邊寫了把 link 貼過來就好 😁

**Reply by yurenju** [reply]
@ChihChengLiang 想要請你發個 PR 登記你的 ethereum address 到 [contributors.csv][1] 裡面！

[1]: https://github.com/lun-dao/LunDAO/blob/main/contributors.csv

**Comment by yurenju** [comment]
@moojing  對了請你發個 PR 登記你的資訊到 [contributors.csv](https://github.com/lun-dao/LunDAO/blob/main/contributors.csv)

**Reply by moojing** [reply]
好的謝謝，我發在這邊 https://github.com/lun-dao/LunDAO/pull/36

**Comment by gasolin** [comment]
如果考慮加入初級文章的話，目前上鏈成本是不是太高啦？

**Reply by yurenju** [reply]
覺得還是要到 layer 2 啦，主網太貴了

**Reply by hydai** [reply]
目前上鏈的成本大概是多少？他會隨著文章的大小有差異嗎？
如果不會我們應該可以用 batch 的方式一批一批上鏈？

**Reply by yurenju** [reply]
@gasolin 你是指獎勵的部分還是文章的部分呀？我目前的想像是：
- 獎勵：在 Layer 2 上面，會在 #11 討論
- 文章：可能是在 IPFS 上面，但是網頁跟 github 都會留存一份

IPFS 就我的理解應該是可以作為備援方案，主要還是保存在 Github 上面，大家如果有 clone 過理論上自己的電腦都會留存一份。

**Comment by iisaint** [comment]
社群的成員經常是來來去去的，必須保持一定的流動性才能永續發展，
因此我也贊成納入初級文章，如此新成員的加入門檻才不至於太高。

至於何謂初級文章？我認為應該聚焦在『原創性 』，
只要是原創文章就有收藏的價值，也可過濾掉市面上千篇一律的入門文章。

**Reply by moojing** [reply]
你這樣的講法讓我想到，如果有某一群定期產出優良文章的人，是不是可以配給他的一個特別的身份，而這些身份可能不是公開的，只能讓社群忠誠使用者看到（也可訂其他條件），這樣也可以進一步增加社群的忠誠度和吸引力。

**Reply by yurenju** [reply]
我比較希望社群的運作盡量公開一些，所以不公開的做法看可不可以避免。

但是我倒是有想到是不是可以做專題報導的形式，像是 @KimiWu123 發起的 zk 讀書會累積了很多 zk 相關的知識，是不是可以邀請他們的成員討論一下要寫哪些文章作為一個系列，可能前面有稍微簡單一些的入門，後面帶著幾篇更深入的分析。我們這邊可以從文字編輯的角度去協助，不管是社群編輯協助看過文章，或是有機會可以請比較專業的編輯幫忙潤稿都是一個方式。

**Reply by KimiWu123** [reply]
可以喔！之前請大家寫文章，大家都沒什麼動力（後來我也是ＸＤ），我覺得有獎勵制度是個好機會

**Reply by yurenju** [reply]
但我覺得最大的困難還是要讓 LUN 產生價值，雖然放了流動性剛開始一定會有價，但是如果只有人賣沒有人買的時候，最終就會很快就跌成💩...

**Reply by gasolin** [reply]
> 是不是可以配給他的一個特別的身份，而這些身份可能不是公開的，只能讓社群忠誠使用者看到（也可訂其他條件），這樣也可以進一步增加社群的忠誠度和吸引力。

也可以用 https://collab.land/ bot 來指定某些 discord channel 只對擁有某些 NFT 或 token 的人開放
